test/helpers/test-server.js

```diff
@@ -27,11 +27,12 @@ function startFullSetup(config, options, done) {
     console.warn('Using the default port for testing is not recommended');
     port = 8080;
   }
-  const host = Object.prototype.hasOwnProperty.call(options, 'host')
+
+  options.host = Object.prototype.hasOwnProperty.call(options, 'host')
     ? options.host
     : 'localhost';
 
-  server.listen(port, host, (err) => {
+  server.listen(port, options.host, (err) => {
```

Comment:
hm, why we change it?

Reply:
I don't exactly recall why but some test cases were failing throwing `different hosts` error. 

We might not need it anymore, I will check if we still need this change.

Reply:
I see for the cases when `host` is `undefined` or not `specified` in config. We can directly pass `options.host` without defaulting to `localhost` above. I will push an update.

Reply:
Done